src/main/java/org/springframework/samples/petclinic/PetClinicRuntimeHints.java

```diff
@@ -24,6 +24,8 @@ public class PetClinicRuntimeHints implements RuntimeHintsRegistrar {
 	@Override
 	public void registerHints(RuntimeHints hints, ClassLoader classLoader) {
 		hints.resources().registerPattern("db/*"); // https://github.com/spring-projects/spring-boot/issues/32654
+		hints.resources().registerPattern("messages/*");
```

Comment:
I don't think any changes to this file should be needed. Seems to work for me anyway.